rules/method-1-using-project-reference-assemblies-within-vsnet/rule.md

```diff
@@ -0,0 +1,25 @@
+---
+type: rule
+title: "Method #1 - Using Project Reference Assemblies within VS.NET"
+uri: method-1-using-project-reference-assemblies-within-vsnet
```

Comment:
What is this rule 😅 @bradystroud? 

@fenix2222 can we change the uri to be a bit smaller 🙏